packages/dialtone-tokens/package.json

```diff
@@ -4,7 +4,10 @@
   "description": "Design tokens for Dialtone.",
   "type": "module",
   "scripts": {
-    "build": "node ./build.js && vite build",
+    "build": "run-s clean build:main build:typescript",
+    "build:main": "node ./build.js && vite build",
+    "build:typescript": "tsc",
+    "clean": "rm -rf ./dist",
```

Comment:
This is going to conflict with my recent changes.

The typescript declaration files are already being exported on dist/themes/types but the export was wrong, will fix it on my next commit to #505

Reply:
just took your changes and discarded mine on conflict, seems to work well.